app/test_main.py

```diff
@@ -1,3 +1,47 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatDogAge:
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (25, 25, [2, 2]),
+            (28, 29, [3, 3]),
+            (32, 34, [4, 4]),
+        ],
+        ids=[
+            "14 cat/dog years should convert into 0 human age.",
+            "15 cat/dog years should convert into 1 human age.",
+            "23 cat/dog years should convert into 1 human age.",
+            "25 cat/dog years should convert into 2 human age.",
+            "28/29 cat/dog years should convert into 3 human age.",
+            "32/34 cat/dog years should convert into 4 human age.",
+        ]
+    )
+    def test_get_human_age(
+            self,
+            cat_age: int,
+            dog_age: int,
+            expected: list
+    ) -> None:
+        assert get_human_age(cat_age, dog_age) == expected
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age",
+        [
+            (-1, -5),
+            (150, 200),
```

Comment:
Negative numbers, 0, large numbers and numbers different for cat and dog age are all valid values and they should be included in the `test_get_human_age`. They do not raise `TypeError`. Test `get_human_age` against strings, dicts, tuples and other data types, they should raise `TypeError` during `get_human_age` execution.